UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/CallbackService.cs

```diff
@@ -42,5 +54,56 @@ public static bool SaveCallbackRequest(CallbackRequest callbackRequest, string s
             }
 
         }
+
+        public bool SaveCommandCallbackRequest(string subscriptionId, HttpStatusCode? httpStatusCode)
+        {
+            try
+            {
+                CommandCallbackRequest? commandCallbackRequest = s_commandCallbackRequestList.LastOrDefault(a => a.SubscriptionId == subscriptionId);
+
+                if (commandCallbackRequest != null)
+                {
+                    if (httpStatusCode == null)
+                    {
+                        s_commandCallbackRequestList.Remove(commandCallbackRequest);
+                    }
+                    else
+                    {
+                        commandCallbackRequest.HttpStatusCode = httpStatusCode ?? NoContent;
```

Comment:
Ok, The case is **commandCallbackRequest.HttpStatusCode** is not nullable and the HTTP status code is nullable, 

**Now I used type cast hope it will be fine.**